test/e2e/config/default.json

```diff
@@ -0,0 +1,95 @@
+{
+	"calypsoBaseURL": "https://wordpress.com",
+	"explicitWaitMS": 20000,
+	"emailWaitMS": 60000,
+	"mochaTimeoutMS": 160000,
+	"mochaDevDocsTimeoutMS": 1000000,
+	"startBrowserTimeoutMS": 60000,
+	"startAppTimeoutMS": 240000,
+	"afterHookTimeoutMS": 60000,
+	"browser": "chrome",
+	"proxy": "direct",
+	"saveAllScreenshots": false,
+	"slackChannelForScreenshots": "#e2eflowtesting-notif",
+	"neverSaveScreenshots": false,
+	"checkForConsoleErrors": false,
+	"reportWarningsToSlack": false,
+	"closeBrowserOnComplete": true,
+	"highlightElements": false,
+	"sauceConfigurations": {
+		"osx-chrome": {
+			"browserName": "chrome",
+			"platform": "OS X 10.13",
+			"screenResolution": "2048x1536",
+			"version": "64.0"
+		},
+	 	"osx-firefox": {
+			"browserName": "firefox",
+			"platform": "OS X 10.11",
+			"screenResolution": "2048x1536",
+			"nativeEvents": true,
+			"version": "46.0"
+	 	},
+	 	"osx-safari": {
+			"browserName": "safari",
+			"platform": "OS X 10.11",
+			"screenResolution": "2048x1536",
+			"nativeEvents": true,
+			"version": "10"
+		},
+		"win-firefox": {
+			"browserName": "firefox",
+			"platform": "Windows 10",
+			"nativeEvents": true,
+			"version": "46.0"
+		},
+		"linux-firefox": {
+			"browserName": "firefox",
+			"platform": "Linux",
+			"nativeEvents": true,
+			"version": "45.0"
+		},
+		"win-ie11": {
+			"browserName": "Internet Explorer",
+			"platform": "Windows 10",
+			"screenResolution": "2560x1600",
+			"version": "11.285",
+		  	"iedriverVersion": "3.141.0",
+		  	"seleniumVersion": "3.141.0"
+		}
+	},
+	"knownABTestKeys": [
+		"businessPlanDescriptionAT",
+		"skipThemesSelectionModal",
+		"springSale30PercentOff",
+		"jetpackSignupGoogleTop",
+		"cartNudgeUpdateToPremium",
+		"signupAtomicStoreVsPressable",
+		"improvedOnboarding",
+		"privateByDefault",
+		"jetpackFreePlanButtonPosition",
+		"removeUsername",
+		"removeDomainsStepFromOnboarding",
+		"showConciergeSessionUpsell",
+		"showConciergeSessionUpsellNonGSuite",
+		"builderReferralThemesBanner",
+		"gSuitePlan",
+		"domainSearchButtonStyles"
+	],
+	"overrideABTests": [
+		[ "businessPlanDescriptionAT_20170605", "original" ],
+		[ "skipThemesSelectionModal_20170904", "show" ],
```

Comment:
Same here! All this could go!

https://github.com/Automattic/wp-calypso/blob/417ea8219d37955b5b7e842dc9ba7e7a73097c17/client/lib/abtest/active-tests.js#L9

(not a change request, just a bonus to pulling e2e in)